lib/mrsk/cli/main.rb

```diff
@@ -17,9 +17,6 @@ def deploy
       invoke_options = deploy_options
 
       runtime = print_runtime do
-        say "Ensure curl and Docker are installed...", :magenta
-        invoke "mrsk:cli:server:bootstrap", [], invoke_options
-
```

Comment:
No longer need slow/noisy per-deploy bootstrapping if we aren't auto-installing Docker, considering any subsequent Docker usage is equivalent to an "is Docker ready?" check.